src/cpu/aarch64/jit_sve_512_convolution.hpp

```diff
@@ -92,8 +92,7 @@ struct jit_sve_512_convolution_fwd_t : public primitive_t {
         else
             assert(false);
 
-        if (pd()->wants_zero_pad_dst())
-            ctx.memory(DNNL_ARG_DST)->zero_pad(ctx.stream());
+        if (pd()->wants_zero_pad_dst()) ctx.zero_pad_output(DNNL_ARG_DST);
```

Comment:
Hi @Takumi-Honda, the change looks good to me but I just wanted to clarify that we changed the memory initialization process in oneDNN (see option 1 of this [RFC](https://github.com/oneapi-src/oneDNN/tree/rfcs/rfcs/20201119-change-memory-init-behavior)).
The change was fully implemented in 7580a3f4b4572782316dd1059570887054b0a7b0.

In a nutshell, now all implementations can safely assume their inputs to be properly zero padded, but should guarantee that their outputs are properly zero padded. Part of these changes,  the zero pad function was moved (as you noticed) and we introduced a new macro to zero-pad output upon extraction from context (`CTX_OUT_CLEAN_MEM`). Hope this helps.

Reply:
Thank you for letting me know the Commit ID where you changed the behavior and the detail of the change.
I will read the READ.md to understand why the behavior was changed.